core/sail/fts/elasticsearch/src/main/java/org/eclipse/rdf4j/sail/elasticsearch/ElasticsearchIndex.java

```diff
@@ -173,7 +174,7 @@ public void initialize(Properties parameters)
 		throws Exception
 	{
 		super.initialize(parameters);
-		indexName = parameters.getProperty(INDEX_NAME_KEY, DEFAULT_INDEX_NAME);
+		indexName = parameters.getProperty(INDEX_NAME_KEY, UUID.randomUUID().toString());
```

Comment:
This is only set to a random value if the index name is not in the parameters, which it should be if there are no bugs in the logic before this point.

Using a static default index name makes bugs in testing very difficult to identify, and maintainability is very important on a volunteer driven open source project.

An shorter alternative to UUID could be a short static prefix suffixed by an AtomicLong incrementing as necessary, as I did with the Tests to debug them.